package.json

```diff
@@ -2,14 +2,21 @@
   "name": "single-spa-vue",
   "version": "1.10.1",
   "description": "a single-spa plugin for vue.js applications",
-  "main": "lib/single-spa-vue.js",
+  "main": "dist/umd/single-spa-vue.js",
```

Comment:
I kept the main file as a UMD/CJS bundle instead of switching to the new ES bundle and adding `"type": "module"`, thinking that that's probably less likely to cause compatibility issues.